src/app/views/query-runner/QueryInput.tsx

```diff
@@ -66,7 +66,7 @@ export class QueryInput extends Component<IQueryInputProps, any> {
       background: getStyleFor(selectedVerb),
     };
 
-    const httpMethodsToDisplay = (mode === Mode.TryIt && !authenticated ) ? [httpMethods[0]] : httpMethods;
+    const httpMethodsToDisplay = (mode === Mode.TryIt || !authenticated) ? [httpMethods[0]] : httpMethods;
```

Comment:
probably correct but we should consider if users may be confused minus text to say they need to sign in.